src/components/ContactForm.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div class="contact-form">
+    <div class="title-section">
+      <h3 class="title">Join the event</h3>
+      <div class="title-underline"></div>
+    </div>
+    <div class="form-section">
+      <div class="content">
+        <h5 class="subtitle">Do you want to be part of the next Asado Night experience?</h5>
+        <p class="description">Join us by completing the form below. All fields are mandatory.</p>
+        <form class="form">
+          <input class="input" name="name" type="text" v-model.trim="contact.name" placeholder="Name">
+          <input class="input" name="surname" type="text" v-model.trim="contact.surname" placeholder="Surname">
+          <input class="input" name="email" type="email" v-model.trim="contact.email" placeholder="Email">
+          <input class="input" name="phone" type="text" v-model.trim="contact.phone" placeholder="Phone number">
+          <input class="input full-width-input" name="company" type="text" v-model.trim="contact.company" placeholder="Company">
+          <button
+            type="submit"
+            class="submit-button"
+            :class="{'active-submit-button': !checkDisabled}"
+            :disabled="checkDisabled"
+          >
+            Join the event
+          </button>
+        </form>
+      </div>
+      <img src="../assets/img/wine.svg" alt="">
+    </div>
+    <div class="line first-line"></div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ContactForm',
+  data () {
+    return {
+      contact: {
+        name: '',
+        surname: '',
+        email: '',
+        phone: '',
+        company: ''
+      }
+    }
+  },
+  computed: {
+    checkDisabled: function () {
+      const { email, name, surname, phone, company } = this.contact
+      console.log(this.contact)
+      return name === '' || surname === '' || email === '' || phone === '' || company === ''
```

Comment:
I'm not sure, could we do something like `[name, surname, email, phone, company].includes('')` here?